core/deployment/src/main/java/io/quarkus/deployment/Capability.java

```diff
@@ -64,7 +64,11 @@
      */
     SMALLRYE_OPENTRACING,
     SPRING_WEB,
-    SMALLRYE_OPENAPI;
+    SMALLRYE_OPENAPI,
+    /**
+     * Presence of config-yaml
+     */
+    YAML;
```

Comment:
Could we name it CONFIG_YAML instead (and move it to respect the alphabetical order)? That would be in line with the name of the `Feature`.

Bonus point: you can remove the comment.

(we might want to test pure Yaml support at some point)

Reply:
Good idea. Done